channels/views_test.py

```diff
@@ -411,6 +414,38 @@ def test_get_post_not_found(client, logged_in_profile):
     assert resp.status_code == status.HTTP_404_NOT_FOUND
 
 
+def test_get_post_stickied(client, private_channel_and_contributor, reddit_factories, staff_user):
+    """test that stickied posts come back that way"""
+    channel, user = private_channel_and_contributor
+    post = reddit_factories.text_post('just a post', user, channel=channel)
+    client.force_login(staff_user)
+    url = reverse('post-detail', kwargs={'post_id': post.id})
```

Comment:
suggest using `staff_api.sticky()` instead of an API request because we're interested only in testing the `GET` here.